samcli/commands/init/__init__.py

```diff
@@ -286,9 +286,9 @@ def do_cli(
     image_bool = name and pt_explicit and base_image
     if location or zip_bool or image_bool:
         # need to turn app_template into a location before we generate
-        templates = InitTemplates(no_interactive)
+        templates = InitTemplates()
```

Comment:
Why we are not passing `no_interactive` flag anymore?

Reply:
Because prior to this changes `InitTemplates` class did not been use it, it was redundant to pass what we don't use.